content/blog/2025/04/kubewarden-1.24-release.md

```diff
@@ -0,0 +1,126 @@
+  have automation in place that cuts monthly patch releases of each policy to ensure
+  all these fresh dependencies are available to all of you.
+
+To automate this, we have created new parts in our
+[kubewarden/github-actions](https://github.com/kubewarden/github-actions)
+v4.4.4:
+
+1. We added a new reusable workflow, `reusable-release-pr`, that can be
+   triggered on a schedule. When it runs, it checks for changes from the last
+   draft release, and opens a PR with [Updatecli](updatecli.io) if it finds changes. This is done
+   via the new
+   [`open-release-pr`](https://github.com/kubewarden/github-actions/blob/main/.github/workflows/reusable-release-pr.yml)
+   You can find the updatecli policy
+   [here](https://github.com/kubewarden/automation/tree/main/updatecli).
+2. We have a second reusable workflow,
+   [`reusable-release-tag`](https://github.com/kubewarden/github-actions/blob/main/.github/workflows/reusable-release-tag.yml).
+   This workflow checks for PRs that the previous workflow opened, and, once the
+   maintainers merge the PR, tags the policy so it gets released.
+
+This process works for normal policy repos and monorepos, such as
+[the one](https://github.com/kubewarden/rego-policies-library) that holds all our
+rego policies.
+
+## cel-policy v1.3.1 released
+
+The latest release of the cel-policy comes with the [Encoder CEL
+extension](https://pkg.go.dev/github.com/google/cel-go/ext#Encoders), which
+allows users to encode/decode base64 strings. In addition, it contains bumps of
+`github.com/google/cel-go` to v0.23.2, plus `k8s.io/apiserver` and
+`k8s.io/apimachinery` to v1.33.0.
+
+## Bugs, minor features and maintenance bumps
+
+As usual, we have kept busy by paying back technical debt for a better future.
+
+### Audit scanner
+
+The Audit Scanner logs have now moved from consuming [zerolog](https://github.com/rs/zerolog) to
+[slog](https://pkg.go.dev/log/slog). The latter is a structured logging library
+that is part of Go's official standard library. We did this change to
+reduce our dependency tree.
+If you are consuming the logs, there are no differences in output.
+Thanks to Dharmit Shah for the contribution!
+
+### Policy server & kwctl
+
+The policy-server code that deals with [policy groups](https://docs.kubewarden.io/howtos/policy-groups) has been moved the
+[policy-evaluator library](https://github.com/kubewarden/policy-evaluator/).
+This refactoring paves the way for future improvements to `kwctl`, like being able
+to use `kwctl run` to evaluate policy groups.
+
+Finally, when the Kubewarden stack is [hardened using
+mTLS](https://docs.kubewarden.io/reference/security-hardening/webhooks-hardening#require-the-kubernetes-api-server-to-authenticate-to-the-webhook),
+the Policy Server instances will
+log all the connection attempts done by client that cannot be identified/trusted.
+This change didn't make the cut of 1.24, but will surely be in 1.25.
+Thanks to Kirat for the contribution!
```

Comment:
We have to remove these lines, they are not about the mTLS logging improvement. This change is part of 1.24, see "bug fixes" section of  the [changelog](https://github.com/kubewarden/policy-server/releases/tag/v1.24.0).

```suggestion
```